application/mqttpubsub/backend.go

```diff
@@ -98,3 +95,11 @@ func (b *Backend) txPayloadHandler(c *mqtt.Client, msg mqtt.Message) {
 
 	b.txPayloadChan <- txPayload
 }
+
+// Subscribe to Topic
+func (b *Backend) onConnected(c *mqtt.Client) {
+	log.WithField("topic", "application/+/node/+/tx").Info("application/mqttpubsub: subscribing to tx topic")
+	if token := b.conn.Subscribe("application/+/node/+/tx", 0, b.txPayloadHandler); token.Wait() && token.Error() != nil {
+		log.WithField("error", "application/+/node/+/tx").Info("application/mqttpubsub: subscribing to tx topic error: %s", token.Error())
```

Comment:
I think ``Info`` must be ``Errorf`` at least (note as well the trailing ``f`` when the log string contains formatting, else it would just print ``"%s"`` :wink: ). What happens when the subscribe fails? Maybe it would be good to put a loop around it, so the server will re-try until subscribed? E.g.

```go
var subscribed bool
for !subscribed {
    if token := b.conn.Subscribe(....); token.Wait() && token.Error() != nil {
        log.WithField("topic", "application/+/node/+/tx").Errorf("application/mqttpubsub: subscribing to topic failed: %s", token.Error())
        time.Sleep(time.Second)
        continue
    }
    subscribed = true
}
```